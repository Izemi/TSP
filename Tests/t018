#!/bin/bash
# Nearest + Insert

trap "/usr/bin/killall -q -u $USER ./TSP 2>/dev/null" 0 1 2 3 9 15
trap "/bin/rm -f $STDERR" 0 1 2 3 9 15
if [ ! -x ./TSP ]; then
  echo './TSP is missing or not executable'
  echo './TSP is missing or not executable' 1>&2
  exit 1
fi


$BIN/run -stderr=/dev/null ./TSP $WHERE/ne_6.dat -nearest -insert HVN ALB MHT BDL ORH PVD < /dev/null
