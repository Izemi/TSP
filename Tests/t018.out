-nearest  :       734.16 HVN ALB MHT PVD ORH BDL HVN
-insert   :       721.47 HVN BDL ALB ORH MHT PVD HVN
